src/ResourceManager/NotificationHubs/Commands.NotificationHubs.Test/ScenarioTests/NotificationHubServiceTests.ps1

```diff
@@ -0,0 +1,565 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Get valid resource group name
+#>
+function Get-ResourceGroupName
+{
+    return "RGName-" + (getAssetName)
+}
+
+<#
+.SYNOPSIS
+Get valid NotificationHub name
+#>
+function Get-NotificationHubName
+{
+    return "NotificationHub-" + (getAssetName)
+}
+
+<#
+.SYNOPSIS
+Get valid Namespace name
+#>
+function Get-NamespaceName
+{
+    return "Namespace-" + (getAssetName)
+}
+
+<#
+.SYNOPSIS
+Tests NotificationHub Namespace Create List Remove operations.
+#>
+function Test-CRUDNamespace
+{
+    # Setup    
+    $location = "South Central US"
+	
+	Write-Debug "Create resource group"
+    $resourceGroupName = Get-ResourceGroupName
+    New-AzureRmResourceGroup -Name $resourceGroupName -Location $location -Force
+	Write-Debug "ResourceGroup name : $resourceGroupName" 
```

Comment:
please align all the tabs in this file. You'll need to put 4 spaces instead of tab everywhere.